products/almalinux.md

```diff
@@ -20,7 +23,17 @@ auto:
       regex: '^Distribution Release: AlmaLinux OS (?P<major>\d).(?P<minor>\d+)$'
 
 releases:
+-   releaseCycle: "10"
```

Comment:
@captn3m0 do we accept beta versions as their release start date ? even do we add any not released but beta in endoflife ?

Reply:
Thanks @chenrui333, @usta, and @captn3m0! I didn't see anything specifically disallowing betas in the guides, but I know that things like this aren't always documented perfectly. If betas aren't allowed, I'm happy to update my PR to just include the additional information for now, and then come back when we release AlmaLinux 10 stable next year.

Reply:
Beta is just beta not release cycle :D so lets make this for info part and we prepare a new one for new one released @bennyvasquez

Reply:
@bennyvasquez thank you